test/hariko/resource/parse_spec.js

```diff
@@ -12,181 +12,215 @@ describe('hariko/resource/parser', function () {
+                }
+              },
+              "response": {
+                "statusCode": 200,
+                "headers": [{name: 'Content-Type', value: 'application/json'}],
+                "body": "    []\n",
+                "data": []
               }
             },
-            "response": {
-              "statusCode": 200,
-              "headers": [{name: 'Content-Type', value: 'application/json'}],
-              "body": "    []\n",
-              "data": []
-            }
-          },
-          {
-            "file": "api/item/item_id-GET.json",
-            "request": {
-              "method": "GET",
-              "uri": {
-                "path": "/api/item/:item_id",
-                "template": "/api/item/{item_id}",
-                "queries": []
+            {
+              "file": "api/item/item_id-GET.json",
+              "request": {
+                "method": "GET",
+                "uri": {
+                  "path": "/api/item/:item_id",
+                  "template": "/api/item/{item_id}",
+                  "queries": []
+                }
+              },
+              "response": {
+                "statusCode": 200,
+                "headers": [{name: 'Content-Type', value: 'application/json'}],
+                "body": "    []\n",
+                "data": []
               }
             },
-            "response": {
-              "statusCode": 200,
-              "headers": [{name: 'Content-Type', value: 'application/json'}],
-              "body": "    []\n",
-              "data": []
+          ],
+          warnings: [
+            {
+              "code": 5,
+              "message": "unexpected header block, expected a group, resource or an action definition, e.g. '# Group <name>', '# <resource name> [<URI>]' or '# <HTTP method> <URI>'",
+              "location": [
+                {"index": 686, "length": 7}
+              ]
+            },
+            {
+              "code": 6,
+              "message": "action is missing a response",
+              "location": [
+                {"index":693,"length":26}
+              ]
             }
-          },
-        ],
-        warnings: [
-          {
-            "code": 5,
-            "message": "unexpected header block, expected a group, resource or an action definition, e.g. '# Group <name>', '# <resource name> [<URI>]' or '# <HTTP method> <URI>'",
-            "location": [
-              {"index": 686, "length": 7}
-            ]
-          },
-          {
-            "code": 6,
-            "message": "action is missing a response",
-            "location": [
-              {"index":693,"length":26}
-            ]
-          }
-        ]
+          ]
+        });
       });
     });
+    describe('when linebreak by \r', function () {
+      var data;
+      beforeEach(function () {
+        data = parser.parse(`\r# GET /api/message\r\r+ Response 200 (application/json)\r\r  + Body\r\r            {message: \"hello\"}\r\r`);
```

Comment:
__[JSHint]__

'template literal syntax' is only available in ES6 (use 'esversion: 6').


<!-- Comment by [SideCI](https://sideci.com) -->